src/Microsoft.AspNet.Hosting/Samples_WebApplication_delete_me.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Hosting
+{
+    internal class SampleApplications
+    {
+        // Template
+        public void Main_ProjectTemplate(string[] args)
+        {
+            var config = WebApplicationConfiguration.GetDefault(args);
+
+            var application = new WebApplicationBuilder()
+                .UseConfiguration(config)
+                .Build();
+
+            application.Run();
+        }
+
+        public void Main_ConfiguringAddresses(string[] args)
+        {
+            var config = WebApplicationConfiguration.GetDefault(args);
+
+            var application = new WebApplicationBuilder()
+                .UseConfiguration(config)
+                .Build();
+
+            var addresses = application.GetAddresses();
+            addresses.Add("http://localhost:5000");
+            addresses.Add("http://localhost:5001");
+
+            application.Run();
+        }
+
+        // Manual hosting and blocking
+        public void Main_ControlOverWaiting(string[] args)
+        {
+            var config = WebApplicationConfiguration.GetDefault(args);
+
+            var application = new WebApplicationBuilder()
+                .UseConfiguration(config)
+                .Build();
+
+            using (application.Start())
+            {
+                Console.ReadLine();
+            }
+        }
+
+        public void Main_FullControl()
+        {
+            var builder = new WebApplicationBuilder()
+                .UseServerFactory("Microsoft.AspNet.Server.Kestrel") // Set the server manually
+                .UseEnvironment("Development")
+                .UseWebRoot("public")
+                .UseLoggerFactory(loggerFactory => // Configure logging
+                {
+                    loggerFactory.AddProvider(new MyHostLoggerProvider());
+                })
+                .UseServices(services =>
+                {
+                    // Configure services that the application can see
+                    services.AddSingleton<IMyCustomService, MyCustomService>();
+                })
+                .UseStartup(app =>
```

Comment:
That sounds like a good change. Configure sounds better. I'll change it to Configure and ConfigureServices for the callbacks to see how it looks.